src/highlight.js

```diff
@@ -45,14 +45,14 @@ export default class Highlight extends Plugin {
  *			model: 'pinkMarker',
  *			class: 'marker-pink',
  *			title: 'Pink Marker',
- *			color: '#fc7999',
+ *			color: 'var(--ck-marker-pink)',
  *			type: 'marker'
  *		}
  *
  * @typedef {Object} module:highlight/highlight~HighlightOption
  * @property {String} title The user-readable title of the option.
  * @property {String} model The unique attribute value in the model.
- * @property {String} color The color used for the highlighter. It should match the `class` CSS definition.
+ * @property {String} color The name of CSS variable used for the highlighter. It should match the `class` CSS definition.
```

Comment:
I think this is wrong for 2 reasons:
* We must still allow developers to use colors in hex/rgba/hsl.
* Even if the above wasn't true, we have an entire `var()` [statement](https://github.com/ckeditor/ckeditor5-highlight/pull/13/files#diff-5d9139cb5fca768116544c986e8c2c8cR48) instead of the name of the variable. 

We should make it clear that all hex/rgba/hsl/var() are possible.